policyengine_us/parameters/gov/contrib/harris/rent_relief_act/rent_relief_credit/rate/gross_income.yaml

```
@@ -0,0 +1,11 @@
+description: Kamala Harris proposed to provide a rent relief credit if the rent payments exceed this percenatge of the housheolds gross income.
```

Could you also rethink the parameter path/name/label?
```suggestion
description: Kamala Harris proposed to provide a rent relief credit to households who spend more than this share of income on rent.
```

---

adjusted but open for suggestions